classes/Page/Page.mjs

```diff
@@ -115,11 +119,43 @@ export default class Page {
     }
 
     async delete() {
-        if(this.#tinyAction === 'update') {
+        if (this.#tinyAction === 'update') {
             // associated Annotations in RERUM will be left intact
-            await databaseTiny.remove(this.id)
-            .catch(err => false)
+            await databaseTiny.remove(this.id).catch(err => false)
         }
         return true
     }
+
+    async #recordModification(userId) {
```

Comment:
This seems to be tryingt o do a lot of work that doesn't need to be done except in Project.

Reply:
Maybe this should eb somethign passed onto User?